lib/hedwig/transports/tcp.ex

```diff
@@ -153,7 +154,7 @@ defmodule Hedwig.Transports.TCP do
     {:noreply, state}
   end
 
-  defp handle_data(_socket, data, state) do
+  defp handle_data(_socket, data, state) when data != "</stream:stream>" do
```

Comment:
I think I would prefer that this be handled within a `case` of the parsed data rather than a guard.